payments_service/settings/__init__.py

```diff
@@ -1,5 +1,9 @@
+import os
 from .base import *  # noqa
 try:
-    from .local import *  # noqa
+    if os.environ.get('UNDER_TEST') == '1':
+        print 'Not importing local settings under test'
```

Comment:
meh, I'd like to avoid it. I think mock patching is probably enough. I suppose we'll see if something breaks from an unexpected default.